perf_benchmark/benchmark_input_stream_translator_integration_test.cc

```diff
@@ -197,6 +242,167 @@ TEST(BenchmarkInputStreamTest,
   }
 }
 
+//
+// Start of gRPC to JSON integration benchmark tests
+//
+TEST(BenchmarkInputStreamTest,
+     IntegrationWithGrpcResponseTranslatorBytesProto) {
+  // Proto message containing "Hello World!"
+  BytesPayload proto;
+  pb::TextFormat::ParseFromString(R"(payload : "Hello World!")", &proto);
+  // "SGVsbG8gV29ybGQh" is the base64 encoded string of "Hello World!"
```

Comment:
Yeah, it was confusing for me as well at the beginning.
This is the expected behaviour because the JSON representation for bytes protobuf field is base64 encoded. When we set the field value from a proto object in c++, we don't need to consider the underlying encoding in its JSON representation. The transcoder will automatically convert the string to a base64 encoded string when it sees the field is `bytes` typed.

Quote from https://developers.google.com/protocol-buffers/docs/proto3#json
`JSON value will be the data encoded as a string using standard base64 encoding with paddings. Either standard or URL-safe base64 encoding with/without paddings are accepted.`